README.md

```diff
@@ -52,10 +52,60 @@ If you prefer the programming ligature so far, I recommend Fira Code because it
 
 ## Installation
 
+### macOS
+
+1. Download font files
+1. Unzip the archive and install the font:
+1. Drag & Drop the font file to Font Book (or the other font management app)
+
+### Windows
+
 1. Download font files
 1. Unzip the archive and install the font:
-    - macOS: Drag & Drop the font file to Font Book (or the other font management app)
-    - Windows: right-click any of them, then pick `Install` from the menu
+1. right-click any of them, then pick `Install` from the menu
+
+### Debian
+
+The other Linux distributions might be same.
+
+1. clone repository
+
+    ```sh
+    git clone https://github.com/0xType/0xProto.git
+    ```
+
+1. build fonts
```

Comment:
This step is not necessary because pre-built font files are in `/font` directory.